nerfstudio/data/dataparsers/depth_dataparser.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 The Nerfstudio Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Depth data parser."""
```

Comment:
Given that this is following the same format at the nerfstudio data, what are your thoughts on adding a `if "depth_file_path" in frame` logic to the nerfstudio-dataparser (similar to what we do with masks)

Reply:
Great idea! Went went this approach instead.